spyt-package/src/main/python/spyt/spec.py

```diff
@@ -43,6 +43,7 @@ class SparkDefaultArguments(object):
     @staticmethod
     def get_params():
         return {
+            "spyt_distributions": { "yt_root": "//home/spark" },
```

Comment:
I'd rename this parameter to `spyt_distribution` or `spyt_distribution_root` and set it to `//home/spark` by default because it is the only sensible value in this structure for now.

Reply:
I made this entity parameter nested so it would be possible to support other way of distributing YT, for example – `"spyt_distributions": {"local_path": "/opt/spyt" }`. I don't think that this should be a flat string parameter.

Reply:
But in this pull request you don't support it. I think it should be kept as simple as possible, and if we decide to support other ways of specifying distribution (btw, is there a need for this?) we just refactor it

Reply:
I've meant that if we will want to support it we will have to add a new conflicting parameter entity instead of extending an existing one. That's why I didn't want to make it a flat string.

Reply:
But what's the possibility of such situation? I think it's almost none

Reply:
I believe that in the future we will most likely want to support ways of distributing SPYT different from a YT catalog, for example in Docker images, like the Spark does.